sandy/njoy.py

```diff
@@ -755,10 +757,30 @@ def _errorr_input(endfin, pendfin, gendfin, errorrout, mat,
     0 293.6 /
     1 33 1/
     1 0 /
-    2 /    
+    2 /
+
+    Test of wrong mt number:
+    >>> with pytest.raises(SyntaxError): sandy.njoy._errorr_input(20, 21, 0, 22, 9237, mt=455)
+    >>> print(sandy.njoy._errorr_input(20, 21, 0, 22, 9237, mt=[102,455]))
```

Comment:
Why don't we get a error/warning here?

Reply:
I would just raise a error right away if one of the banned_mt is found in the mt list

Reply:
This assumes that the user knows how NJOY is programmed in sandy, which is not always the case. The current programming only raises an error if we are absolutely sure that NJOY is going to fail.